worm_war/sources/board.h

```diff
@@ -11,6 +12,7 @@ enum WormType {
 };
 
 class Board {
+    std::mutex mtx;
```

Comment:
I think it should be after `private:`. I know that is it private by default, but the convention is that you always give that specifier for readability.